explorer/interpreter/action_stack.h

```diff
@@ -104,7 +109,19 @@ class ActionStack {
   auto UnwindPast(Nonnull<const Statement*> ast_node,
                   Nonnull<const Value*> result) -> ErrorOr<Success>;
 
-  void Pop() { todo_.Pop(); }
+  void Pop() {
+    todo_.Pop();
+    if (trace_stream_->is_enabled()) {
+      *trace_stream_ << "+++ stack: [ " << *this << " ]\n";
+    }
+  }
```

Comment:
It looks like this function isn't being called -- in action_stack.cpp, `todo_.Pop()` is called directly instead of using this function.